### DISCUSSION TITLE
React.js Component

### DISCUSSION THREAD

**Discussion by masteryodaa** [geoman-io/leaflet-geoman#1030] [Questions & Help] [answered]
are there any available integrations for it in reactjs as components?

**Comment by Falke-Design** [comment, marked as answer]
Here are some related links:
- https://codesandbox.io/s/angry-babbage-cns52?file=/src/App.tsx
- https://codesandbox.io/s/394eq
- https://github.com/geoman-io/leaflet-geoman/issues/417#issuecomment-627242958

**Reply by Marteyo** [reply]
Hi, thanks for the answer.
This code helps me [https://codesandbox.io/s/394eq](https://codesandbox.io/s/394eq), but sometimes when I click on a polygon makes an error:
```
SecurityError
The operation is insecure.
```
On the other hand, is there any code with version 4.0 of react-leaflet?
Thank you very much!
